packages/storage-evm/scripts/base-mainnet-contracts-scripts/set-mint-transfer-whitelist.ts

```
@@ -0,0 +1,181 @@
+  const fileContent = fs.readFileSync(addressesPath, 'utf8');
+
+  const addresses: Record<string, string> = {};
+
+  // Extract contract addresses using regex
+  const patterns = {
+    HyphaToken: /HyphaToken deployed to: (0x[a-fA-F0-9]{40})/,
+  };
+
+  for (const [key, pattern] of Object.entries(patterns)) {
+    const match = fileContent.match(pattern);
+    if (match && match[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)